src/seedu/addressbook/data/person/Person.java

```diff
@@ -10,12 +10,15 @@
  */
 public class Person implements ReadOnlyPerson {
 
+	private static int nextSequenceNumber = 1;
```

Comment:
Notice that your indentation mechanism uses tabs for indentation. 
The coding standards we follow require using 4 spaces for indentation.
Tip: you can set Eclipse to do this for you, check out how!